src/components/Footer.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "../index.css";
+
+function Footer() {
```

Comment:
In footer we can have contact or something like that. But for cleaner look it can be removed completely if needed. All information can be placed on landing page or separate pages.

Reply:
Visually footer completes look along with header and gives users feedback that there isn't any more content on the page. In the future development footer will be upgraded with more features.